### ISSUE TITLE
Make use of branches

### ISSUE THREAD

**Issue by mvdbeek** [ARTbio/GalaxyKickStart#38]
@fabiorjvieira 
I see you're heavily modifying the dev branch, which is good obviously.
But I think you could really profit from using a branch per feature or bug.
This would avoid ending up with a huge incompatible merge, like it is now.
<img width="1644" alt="screen shot 2015-12-04 at 17 43 40" src="https://cloud.githubusercontent.com/assets/6804901/11595175/6e9b4c60-9aae-11e5-846c-881db9963fb3.png">

Also, this would make it much easier for @drosofff and me to evaluate and comment on the things you do. 
Plus you can do regular updates to the master branch, so we can work with it.
For example the changes you are working on for persisting postgresql seem very complex considering a bind-mount should suffice (I could be wrong, but discussing about this in isolation is much easier). Also consider that this is a very standard job, so tested & proven best-practice solutions are likely out there.

In summary, everyone wins if you work with one branch per feature (and maybe create an issue before working on a feature?).
Looking forward to test your changes ;).
